### PR TITLE
Update migration-tidb-faq.md

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added or deleted? (Required)

 update FAQ from oncall

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@lance6716 Please involve a technical review~

File: faq/migration-tidb-faq.md [hunk 1]
```diff
@@ -12,6 +12,20 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
 
 TiDB 支持绝大多数 MySQL 语法，一般不需要修改代码。
 
+### 导入导出速度慢，且日志中出现大量重试、EOF 错误
```
Comment (lichunzhu) [inline faq/migration-tidb-faq.md:15]: 能不能写下是 **什么日志** 中出现大量重试 错误？

File: faq/migration-tidb-faq.md [hunk 2]
```diff
@@ -12,6 +12,20 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
...
+
+```
+# on server node
+iperf3 -s
+
+# on client node
+iperf3 -c <server-IP>
+```
+
+检查带宽、延迟、重传等指标是否有异常。
```
Comment (lichunzhu) [inline faq/migration-tidb-faq.md:27]: 什么样的结果正常，什么样的结果异常？

Comment (lance6716) [reply faq/migration-tidb-faq.md:27]: 客户自己买的线路是什么指标不好概括

Comment (glorv) [reply faq/migration-tidb-faq.md:27]: 这个 faq 内容有点短吧，如果不是硬件网络资源的问题然后呢？

**Comment by TomShawn** [comment]
@glorv @lichunzhu PTAL again

File: faq/migration-tidb-faq.md [hunk 3]
```diff
@@ -12,6 +12,20 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
 
 TiDB 支持绝大多数 MySQL 语法，一般不需要修改代码。
 
+### 导入导出速度慢，且各组件日志中出现大量重试、EOF 错误
+
+首先怀疑网络问题，使用相关工具排查网络连通状况。下面是使用 [iperf](https://iperf.fr/) 进行排查的例子
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:17]:
```suggestion
可首先考虑是否存在网络问题，使用相关工具排查网络连通状况。以下示例使用 [iperf](https://iperf.fr/) 进行排查：
```

File: faq/migration-tidb-faq.md [hunk 4]
```diff
@@ -12,6 +12,20 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
...
+
+```
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:19]:
```suggestion
```shell
```

File: faq/migration-tidb-faq.md [hunk 5]
```diff
@@ -12,6 +12,20 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
...
+# on server node
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:20]:
```suggestion
# 在服务器节点上执行
```

File: faq/migration-tidb-faq.md [hunk 6]
```diff
@@ -12,6 +12,20 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
...
+iperf3 -s
+
+# on client node
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:23]:
```suggestion
# 在客户端节点执行
```

File: faq/migration-tidb-faq.md [hunk 7]
```diff
@@ -12,6 +12,20 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
...
+iperf3 -c <server-IP>
+```
+
+检查带宽、延迟、重传等指标是否有异常。
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:27]: 这句话和上面的示例有什么关系呢？

**Comment by lance6716** [comment]
@glorv @lichunzhu PTAL again

**Comment by glorv** [comment]
LGTM

File: faq/migration-tidb-faq.md [hunk 8]
```diff
@@ -12,6 +12,47 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
...
+
+在没有其他逻辑出错的情况下，重试、EOF 可能是由网络问题引起的，建议首先使用相关工具排查网络连通状况。以下示例使用 [iperf](https://iperf.fr/) 进行排查：
+
+```shell
+# 在重试、EOF 错误的服务器端节点执行
+iperf3 -s
+
+# 在重试、EOF 错误的客户端节点执行
+iperf3 -c <server-IP>
+```
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:25]:
```suggestion

+ 在出现重试、EOF 错误的服务器端节点执行以下命令：

    {{< copyable "shell-regular" >}}
    
    ```shell
    iperf3 -s
    ```

+ 在出现重试、EOF 错误的服务器端节点执行以下命令：

    ```shell
    iperf3 -c <server-IP>
    ```

```

File: faq/migration-tidb-faq.md [hunk 9]
```diff
@@ -12,6 +12,57 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
 
 TiDB 支持绝大多数 MySQL 语法，一般不需要修改代码。
 
+### 导入导出速度慢，各组件日志中出现大量重试、EOF 错误并且没有其他错误
+
+在没有其他逻辑出错的情况下，重试、EOF 可能是由网络问题引起的，建议首先使用相关工具排查网络连通状况。以下示例使用 [iperf](https://iperf.fr/) 进行排查：
+
+
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:19]:
```suggestion
```

File: faq/migration-tidb-faq.md [hunk 10]
```diff
@@ -12,6 +12,57 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
...
++ 在出现重试、EOF 错误的服务器端节点执行以下命令：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    iperf3 -c <server-IP>
+    ```
+
+
+下面是一个网络连接良好的客户端节点的输出
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:37]:
```suggestion
下面是一个网络连接良好的客户端节点的输出：
```

File: faq/migration-tidb-faq.md [hunk 11]
```diff
@@ -12,6 +12,57 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
...
+
+```
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:39]:
```suggestion
```shell
```

File: faq/migration-tidb-faq.md [hunk 12]
```diff
@@ -12,6 +12,57 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
...
+[  5]   9.00-10.00  sec  21.8 MBytes   183 Mbits/sec
+- - - - - - - - - - - - - - - - - - - - - - - - -
+[ ID] Interval           Transfer     Bitrate
+[  5]   0.00-10.00  sec   208 MBytes   175 Mbits/sec                  sender
+[  5]   0.00-10.00  sec   208 MBytes   174 Mbits/sec                  receiver
+
+iperf Done.
+```
+
+如果输出中网络带宽较低、带宽波动大，会导致各组件出现上述状况。这需要咨询网络服务供应商提升网络质量。
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:62]:
```suggestion
如果输出显示网络带宽较低、带宽波动大，各组件日志中就可能出现大量重试、EOF 错误。此时你需要咨询网络服务供应商以提升网络质量。
```

File: faq/migration-tidb-faq.md [hunk 13]
```diff
@@ -12,6 +12,57 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
...
+
+如果输出中各指标良好，请尝试更新各组件版本。更新后无法解决的问题，请移步 [AskTUG 论坛](https://asktug.com/)寻求帮助。
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:64]:
```suggestion
如果输出的各指标良好，请尝试更新各组件版本。如果更新后仍无法解决的问题，请移步 [AskTUG 论坛](https://asktug.com/)寻求帮助。
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

File: faq/migration-tidb-faq.md [hunk 14]
```diff
@@ -12,6 +12,55 @@ aliases: ['/docs-cn/dev/faq/migration-tidb-faq/']
...
+
++ 在出现重试、EOF 错误的服务器端节点执行以下命令：
+
+    {{< copyable "shell-regular" >}}
+    
+    ```shell
+    iperf3 -s
+    ```
+
++ 在出现重试、EOF 错误的服务器端节点执行以下命令：
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:27]: typo fixed in https://github.com/pingcap/docs-cn/pull/6228.

**Pull request merged**